.github/workflows/testflight.yml

```diff
@@ -0,0 +1,40 @@
+name: Testflight
+
+on:
+  push:
+    tags:
+      - 'v*.*.*-beta*'
+
+jobs:
+  testflight:
+    runs-on: macos-latest
```

Comment:
Looks good, just an idea: Perhaps we could move this job into the iOS workflow along with a workflow dependency on `build` to reuse the setup and (ideally) even the build products. Not a blocker though.

Reply:
Great idea! Added #40 for that.